### PR TITLE
Added

### PR DESCRIPTION
**Pull request by alfari24**
- request from _delusion_ to added it

### CODE REVIEW

File: docs/NoSSL/lavalink-without-ssl.md [hunk 1]
```diff
@@ -234,3 +234,12 @@ Port : 30216
 Password : "somboytiger"
 Secure : false
 ```
+
+### Hosted by @ [𝓓𝓔𝓛𝓤𝓢𝓘𝓞𝓝](https://discord.gg/FqEQtEtUc9)
+Version 3.?
```
Comment (DarrenOfficial) [inline docs/NoSSL/lavalink-without-ssl.md:239]: What version is it? 3.What? 🙁

**Review by DarrenOfficial — CHANGES_REQUESTED** [review]

File: docs/NoSSL/lavalink-without-ssl.md (hunk 1 above)
Comment (alfari24) [reply docs/NoSSL/lavalink-without-ssl.md:239]: if i not wrong, i think he still used version `yt-400-v3`.
let me ask him

Comment (AjieDev) [reply docs/NoSSL/lavalink-without-ssl.md:239]: yes you correct (219dfdd01931edd3d750e2757de23aa2d228b9c9-SNAPSHOT)

Comment (alfari24) [reply docs/NoSSL/lavalink-without-ssl.md]: updated

**Review by DarrenOfficial — APPROVED** [review]

**Pull request merged**